src/uct/tcp/tcp_iface.c

```diff
@@ -32,10 +32,14 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    "option usually provides better performance",
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_nodelay), UCS_CONFIG_TYPE_BOOL},
 
-  {"SNDBUF", "64k",
-   "Socket send buffer size.",
+  {"SNDBUF", "32k",
```

Comment:
Why is 32K better?

Reply:
my bad - it was from testing on ARMs from bgate, no need to have it. returned it back to 64k